zeek

```diff
@@ -92,25 +94,43 @@ init_zeek_cfg() {
 }
```

Comment:
Lines 88-91 will prompt the user to select a capture interface even if the user is only using the `zeek` script for pcap processing. AFAIK, the `.cfg` files are only read by `zeekctrl`, and `zeek` does not need them to run.  Would you be open to splitting up `init_zeek_cfg` into `init_zeekctl_cfg` and `init_zeek_cfg`? That way we can avoid calling `init_zeekctl_cfg` and creating the `.cfg` files in the `readpcap` section of the code.

![image](https://user-images.githubusercontent.com/761220/221723460-6f6d2d72-baff-4ae7-aa42-8a3c0978e23a.png)

Reply:
After checking with Logan, we'll check for readpcap mode and skip creating node.cfg if we're in it.  Changed.